AccountValidation.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: weronikakrzynowek
+ * Date: 30.05.18
+ * Time: 09:18
+ */
+
+class AccountValidation
+{
+    const NAME_PATTERN = "/^[a-zA-Z]*$/";
+    const PASSWORD_PATTERN = "/^(?=.*\d)(?=.*[A-Za-z])[0-9A-Za-z!@#$%]{8,12}$/";
+
+    protected $errorMessages = array();
+    protected $formValues = [];
+
+    public function validateNotEmpty($val, $key)
+    {
+        if(empty($val)) {
+            $this->errorMessages[$key] = "enter require data";
```

Comment:
w przypadku tekstu używamy pojedyńczych apostrofów, cudzysłów używamy tylko w wyjątkowych sytuacjach

Reply:
należy to zastosować dla całego pull requesta